extension/scripts/features/faction-member-info/ttFactionMemberInfo.js

```diff
@@ -0,0 +1,104 @@
+"use strict";
+
+(async () => {
+	if (!isOwnFaction()) return;
+
+	const feature = featureManager.registerFeature(
+		"Member Info",
+		"faction",
+		() => settings.pages.faction.memberInfo,
+		addListener,
+		addInfo,
+		removeInfo,
+		{
+			storage: ["settings.pages.faction.memberInfo"],
+		},
+		() => {
+			if (!hasAPIData() || !hasFactionAPIAccess()) return "No API access!";
```

Comment:
`hasAPIData` is not needed, as that check is also done in `hasFactionAPIAccess`.